frontend/catalyst/__init__.py

```diff
@@ -185,6 +185,7 @@
     "DifferentiableCompileError",
     "CompileOptions",
     "debug",
+    "logging",
```

Comment:
Is this module meant to be used by users?

Reply:
It can be yea. It is provided for users in PennyLane and is allowed to be used for user-facing code (@trbromley  can confirm here).

Reply:
I'm happy to disable this though if preferred, as it just remaps the PennyLane module, which should be interchangeable.

Reply:
I think disabling it in Catalyst and let users to use the one from PL will enable us to market this feature with a unified documentation and guideline across PL stack.

Reply:
Sure. I'll remove this.